src/pages/documentation/mobile-core/rules-engine/index.md

```diff
@@ -55,7 +55,9 @@ On older versions of Experience Platform Mobile SDKs (prior to iOS version 1.6.2
 
```

Comment:
@cdhoffmann This is supported both in iOS and Android. We should remove `(iOS AEP 3.x Only)`

Reply:
(iOS AEP 3.x Only) has been removed.

Reply:
@hbhandal80 can you also include an info block under this title with the text 
```
This feature is available from iOS AEP 3.x and Android 1.11.2
```

Reply:
Note added.